test-common/test-common-api/src/main/java/org/apache/kafka/common/test/api/ClusterInstance.java

```diff
@@ -147,10 +155,44 @@ default <T> T getUnderlying(Class<T> asClass) {
         return asClass.cast(getUnderlying());
     }
 
-    Admin createAdminClient(Properties configOverrides);
+    //---------------------------[producer/consumer/admin]---------------------------//
 
-    default Admin createAdminClient() {
-        return createAdminClient(new Properties());
+    default <K, V> Producer<K, V> producer(Map<String, Object> configs) {
+        Properties props = new Properties();
+        props.putAll(configs);
+        props.putIfAbsent(ProducerConfig.ACKS_CONFIG, "-1");
```

Comment:
Could you please remove this? it would be better to add it if we have observed many similar use cases